smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SmithyTestUtilTypes.kt

```diff
@@ -0,0 +1,14 @@
+package software.amazon.smithy.swift.codegen
```

Comment:
Swift symbols for the SmithyTestUtil module.

(Referenced in the WeatherSDK only.)